Style.css

```diff
@@ -98,3 +98,192 @@ li {
   gap: 10px;
   margin-left: -40px;
 }
+
+.experience {
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  grid-template-columns: 1fr;
+  padding-top: 10%;
+}
```

Comment:
Kindly use the `grid` display property to style the works section, as it is a requirement for this project to use **both** grid and flexbox 🙏.

<img width="756" alt="image" src="https://user-images.githubusercontent.com/37116322/219717166-23e2818f-3e1d-4fa8-80c6-249a6ce030c2.png">